Gaijinizer patch/Map062.txt

```diff
@@ -308,29 +332,29 @@ human emotions ...#MTLed
 だから魔界についてはなにも知らないし、
 逆に人間界のことはよく知ってるの。
 > CONTEXT: Map062/events/5/pages/1/111/Dialogue/enemy4
-Yeah, yes. So I don't know anything about Demon
-Realm, Conversely, I know about the human realm
-well.#MTLed
+Yes, that's right. I don't know anything about the
+demon realm, but I know a lot about the human
+world.#Ron
 > END STRING
 
 > BEGIN STRING
 魔界のやつらは、自分らを『純正種』。
 
 マザーラビットみたいなのを『具現種』と呼んでるな。
 > CONTEXT: Map062/events/5/pages/1/119/Dialogue/other1
-The guys in Demon Realm are "Genuine" for
-themselves. Don't call something like Mother
-Rabbit 'tranformed species'.#MTLed
+The guys in the demon realm call themselves "pure
+demons". Ones like Mother Rabbit are called
+"embodiments".#Ron
```

Comment:
"Embodiments" seems like a rough term here. I'll need to think of some better words for this...